packages/amplify_auth_plugin_interface/lib/src/SignOut/SignOutRequest.dart

```diff
@@ -15,9 +15,17 @@
 
 import './SignOutOptions.dart';
 
+/// Encapsulates parameters for a sign out request
 class SignOutRequest {
+  /// Advanced options for the request
+  ///
+  /// Can be used to specify global sign out
   SignOutOptions? options;
+
+  /// Default constructor
   SignOutRequest({this.options});
```

Comment:
Prefer a more descriptive doc comment for the constructor since this is what is shown when CX goes to create an instance:

<img width="598" alt="Screen Shot 2021-09-22 at 1 19 53 PM" src="https://user-images.githubusercontent.com/24740863/134415915-03c53287-1210-4e13-8c00-89d818d50572.png">

Also, can be final fields/const constructor while we're at it.

```suggestion
/// {@template sign_out_request}
/// Encapsulates parameters for a sign out request
/// {@endtemplate}
class SignOutRequest {
  /// Advanced options for the request
  ///
  /// Can be used to specify global sign out
  final SignOutOptions? options;

  /// {@macro sign_out_request}
  const SignOutRequest({this.options});
```

Reply:
Good call. Updated.